wfl/fit/gap_multistage.py

```diff
@@ -175,23 +156,21 @@ def fit(fitting_configs, GAP_name, params, ref_property_prefix='REF_',
 
     if remote_info is not None and remote_info != '_IGNORE':
         # check if we can skip
-        need_to_recalc = True
+        need_to_fit = True
```

Comment:
Similarly to `ace.py`, could move the for loop into try-except, move returns from lines ~200 into the `try` part, so that exception only happens if the fit is needed. I think that will make the code clearer.

Reply:
I agree.  Refactored fairly substantially in 16cc346.

Reply:
Should be fixed in b5d31ff.  Can you take a look?

Reply:
Looks good to me!